.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
```

Comment:
Travis is failing so I can't merge anyways

Reply:
yes it's a python thing - I got circleCI working last night on another repo - so can try fix this